script.js

```diff
@@ -0,0 +1,109 @@
+const name = document.querySelector('.name');
```

Comment:
your global scope is currently polluted with global variables like name, phone etc. 

Consider using iffy: https://codeburst.io/javascript-what-the-heck-is-an-immediately-invoked-function-expression-a0ed32b66c18